src/coreclr/vm/arm/thunktemplates.S

```diff
@@ -14,6 +14,7 @@ PAGE_SIZE = 4096
 #define DATA_SLOT(stub, field) . - (. - stub##Code) + PAGE_SIZE + stub##Data__##field
 
     LEAF_ENTRY StubPrecodeCode
+        dmb
```

Comment:
Should this barrier still stay here instead of using the cache flushing?

Reply:
Oh, yeah, probably should swap that back.